src/main/java/cn/nukkit/item/randomitem/Fishing.java

```diff
@@ -66,7 +66,6 @@ public static Item getFishingResult(int fortuneLevel, int lureLevel) {
 
     private static float limitRange(float min, float max, float value) {
         if (value >= max) return max;
-        if (value <= min) return min;
-        return value;
+        return Math.max(value, min);
```

Comment:
The whole method could be replaced with clamp (Might be needed to add float version to NukkitMath)